### ISSUE TITLE
Can't scroll overflow:scroll elements

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#10434]
Here is a test case:

```html
<style>

  main {
    height: 300px;
    width: 250px;
    background-image: linear-gradient(to bottom, red, blue);
    overflow: scroll;
  }

  div {
    margin-top: 200px;
    height: 200px;
    width: 200px;
    overflow: scroll;
  }

  p {
    height: 250px;
    background-image: linear-gradient(to bottom, white, black);
    margin: 0;
  }

</style>
<main>
  <div>
    <p></p>
  </div>
</main>
```

**Comment by pcwalton** [comment]
Duplicate of #2742.

This will likely require a lot of work.

**Comment by paulrouget** [comment]
@pcwalton: if it's not something we can realistically fix by June, please let me know, we can probably work around that.

**Comment by paulrouget** [comment]
Re-opening. I don't think this is fixed.

**Comment by pcwalton** [comment]
Yeah, WR doesn't support overflow:scroll scrolling yet.

**Comment by paulrouget** [comment]
> Yeah, WR doesn't support overflow:scroll scrolling yet.

Ok. So I will work around that.

P1 -> P3.

**Comment by pcwalton** [comment]
Fixed by #10717
